lib/nacre/channel_collection.rb

```diff
@@ -14,15 +14,7 @@ def self.request_options
     end
 
     def self.service_name
-      'channel'
-    end
-
-    def self.service_url
-      configuration.resource_url + '/' + 'product-service'
-    end
-
-    def self.url
-      service_url + '/' + service_name
+      'product'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.